CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+## Contributing
+
+This project welcomes contributions and suggestions. Most contributions
+require you to agree to a Contributor License Agreement (CLA) declaring that
+you have the right to, and actually do, grant us the rights to use your
+contribution. For details, visit https://cla.microsoft.com.
+
+When you submit a pull request, a CLA-bot will automatically determine
+whether you need to provide a CLA and decorate the PR appropriately (e.g.,
+label, comment). Simply follow the instructions provided by the bot. You
+will only need to do this once across all repositories using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of
+Conduct](https://opensource.microsoft.com/codeofconduct/). For more
+information see the [Code of Conduct
+FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact
+[opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional
+questions or comments.
```

Comment:
Guess this is a default blurb, should we inform the maintainers of the alias about our adoption of the conduct? In case people actually do email?

Reply:
I copied the CONTRIBUTING.md from pg_auto_failover, mainly for the commit hook installation script. This seemed like a reasonable blurb to keep. I don't think we need to notify the mailinglist of anything, this is sortof the standardized stuff for all microsoft open source projects afaik.